test/controllers/processors/CompanyNameProcessor.test.ts

```diff
@@ -0,0 +1,133 @@
+import 'reflect-metadata';
+
+import Substitute, { Arg } from '@fluffy-spoon/substitute';
+import ApiClient from 'ch-sdk-node/dist/client';
+import CompanyProfileService from 'ch-sdk-node/dist/services/company-profile/service';
+import { CompanyProfile } from 'ch-sdk-node/dist/services/company-profile/types';
+import { assert, expect } from 'chai';
+import { Request } from 'express';
+import { createSubstituteOf } from '../../SubstituteFactory';
```

Comment:
can you change these imported to the shortened version, i.e. `'test/SubstituteFactory'`